### PR TITLE
Project 2 - Milestone 2

### PR DESCRIPTION
**Pull request by John-Arboleda**
This pull request is a clone of [TheNextWeb](https://perma.cc/M5ZV-Q2D6) home page as an exercise for **Grid**, **Resposive Desing** and **background linear gradient.**

- Mobile-first design was implemented. Media Queries modify the layout for Tablet and Desktop view.
- Navbar, Highlights section, lastest funding rounds, and footer were created with flexbox.
- Latest news and Lates deals sections were implemented on Grid.
- Background linear gradient was applied for the highlights section images.

Best,
**John & Francis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,6 +135,71 @@ <h2 class="cover-title">
...
 
+<section class="latest-news">
+  <header>
+    <a class="latest-news-title" href="#">Latest news</a>
+  </header>
+  <div class="latest-article-container">
+    <article class="latest-article">
+      <img class="latest-article-photo" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png" alt="image" width="75" height="75">
+      <header>
+        <a href="#" class="latest-article-title">Want to drive a hydrogen EV in the US? Move to California</a>
```
Comment (wathigo) [inline index.html]: Use a header tag (hx) for the title of the section, that is also a link (a tag)

File: index.html [hunk 2]
```diff
@@ -135,6 +135,71 @@ <h2 class="cover-title">
   </div>
 </div>
 
+<section class="latest-news">
+  <header>
+    <a class="latest-news-title" href="#">Latest news</a>
+  </header>
+  <div class="latest-article-container">
+    <article class="latest-article">
+      <img class="latest-article-photo" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png" alt="image" width="75" height="75">
```
Comment (wathigo) [inline index.html]: Each article has an image and a title (`hx` tag), both of them with a link (`a` tag)

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this milestone.
The mobile-first approach method has been used to implement responsiveness.
Good job on using semantic HTML5 elements too.
Let's fix a few issues before we proceed.

### REQUIRED
* Files that are not related to the project should not be committed. Instead, they should be included in `gitignore` file. Let's delete project un-related files such as `.vscode`.
* See comments for more suggestions :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

File: index.html (hunk 1 above)
Comment (John-Arboleda) [reply index.html]: Are you suggesting a header tag in the title of row 146, which is already inside a header tag?

**Review by voscarmv — CHANGES_REQUESTED** [review]
Impressive work. 💻 👀 ✨ 
- Congratulations! Your project is looking great.
- Excellent progress from very early on! Your page looks extremely close to the original.

## However, there are some changes required 👀 🧰 🔨 🔧 
- The only detail I could find was with the Latest News section articles. The Live Version shows broken images on the articles. Please fix this before moving forward!
image

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by John-Arboleda** [comment]
@voscarmv

@wathigo saw no broken links. After 13 hours waiting for a code review, the links ain't working anymore.

**Comment by voscarmv** [comment]
> @voscarmv
> 
> @wathigo saw no broken links. After 13 hours waiting for a code review, the links ain't working anymore.

We are assigned code reviews randomly by the submissions system and review them as soon as we possibly can. I'm sorry this happened but please kindly fix the issue before moving on.

Project milestones are an unavoidable part of the review process so even if I had not found any issues in your code, you still would have had to pass through milestone 2.

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://john-arboleda.github.io/TNW-clone-page/) shows a good understanding of Responsive Design. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @John-Arboleda and @francisuloko ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by francisuloko** [comment]
Thank you for reviewing and approving our code @umairarshadbutt. Do enjoy the rest of your day. Happy Coding :computer:

**Pull request merged**